driver/driver.hpp

```diff
@@ -198,7 +198,8 @@ inline std::string ParseBaseArg(int argc, char* argv[])
        arg != "layernormfp16" && arg != "layernormbfp16" && arg != "sum" && arg != "sumfp16" &&
        arg != "sumbfp16" && arg != "argmax" && arg != "argmaxfp16" && arg != "argmaxbfp16" &&
        arg != "groupnorm" && arg != "groupnormfp16" && arg != "groupnormbfp16" && arg != "cat" &&
-       arg != "catfp16" && arg != "catbfp16" && arg != "--version")
+       arg != "catfp16" && arg != "catbfp16" && arg != "getitem" && arg != "getitemfp16" &&
```

Comment:
It would be nice if these were grouped by algorithm. I suspect that clang-format is doing this. I'm fairly new to clang-format; is there a way to override it?

Reply:
Yes, clang-format is doing it. We can use clang-format off, but a comment is added.